packages/components/src/components/hds/application-state/media.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import TemplateOnlyComponent from '@ember/component/template-only';
+
+export interface HdsApplicationStateMediaSignature {
+  Element: HTMLDivElement;
+  Blocks: {
+    default: [];
+  };
```

Comment:
For consistency with the other components
```suggestion
  Blocks: {
    default: [];
  };
  Element: HTMLDivElement;
```